src/main/java/in/zapr/druid/druidry/aggregator/DoubleMaxAggregator.java

```diff
@@ -19,17 +19,28 @@
 import lombok.EqualsAndHashCode;
 import lombok.Getter;
 import lombok.NonNull;
+import lombok.Setter;
 
 @Getter
 @EqualsAndHashCode(callSuper = true)
```

Comment:
For excluding `null` fields as now we would have either of `fieldName` or `expression ` as null.
```suggestion
@EqualsAndHashCode(callSuper = true)
@JsonInclude(JsonInclude.Include.NON_NULL)
```